Tensile/TensileCreateLibrary.py

```diff
@@ -1428,6 +1440,7 @@ def splitExtraParameters(par):
   if not os.path.exists(logicPath):
     printExit("LogicPath %s doesn't exist" % logicPath)
 
+
   # Translate GPU targets to filter filenames in Tensile_LOGIC directory
   mapArchitecture = {'all':'_','gfx000':'none', 'gfx803':'r9nano',
         'gfx900':'vega10', 'gfx906:xnack-':'vega20', 'gfx908:xnack-':'arcturus'}
```

Comment:
This conflicts with the new global definition @ line 50. This one includes new xnack flags, which will be needed? 
Can we consolidate these tables and / or put them into common ? 

Also the point of this table seems to be mapping architecture aliases, so would a better name be architectureAliasMap?

Reply:
fixed: This was moved to Common.py.